src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -838,4 +842,32 @@ def load_arguments(self, _):
     with self.argument_context('vm monitor log show') as c:
         c.argument('analytics_query', options_list=['--analytics-query', '-q'], help="Query to execute over Log Analytics data.")
         c.argument('timespan', help="Timespan over which to query. Defaults to querying all available data.")
+
+    with self.argument_context('vm monitor metrics') as c:
+        c.argument('metricnamespace', options_list=['--namespace'],
+                   help='Namespace to query metric definitions for.')
+
+    with self.argument_context('vm monitor metrics tail') as c:
+        from azure.mgmt.monitor.models import AggregationType
+        c.extra('resource_group_name', required=True)
+        c.argument('resource', arg_type=existing_vm_name, help='Name or ID of a virtual machine', validator=validate_vm_name_for_monitor_metrics, id_part=None)
+        c.argument('metadata', action='store_true')
+        c.argument('dimension', nargs='*', validator=validate_metric_dimension)
+        c.argument('aggregation', arg_type=get_enum_type(t for t in AggregationType if t.name != 'none'), nargs='*')
+        c.argument('metrics', nargs='+')
+        c.argument('orderby',
+                   help='Aggregation to use for sorting results and the direction of the sort. Only one order can be specificed. Examples: sum asc')
+        c.argument('top', help='Max number of records to retrieve. Valid only if --filter used.')
+        c.argument('filters', options_list='--filter')
```

Comment:
Can we add multiple filters in one string?

Reply:
It's still a filter even though it support multiple conditions through `and`

Reply:
It's still a good catch. I will update help information. Since for VM, only one condition can be used.